experiments/compositions/composition/dev/tasks/generate-proto

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd ${REPO_ROOT}/experiments/compositions/composition
+
+BUILDDIR=$(pwd)/.build/
+BINDIR=${BUILDDIR}/bin/
+mkdir -p ${BINDIR}
```

Comment:
`wget` is used here, but it's generally better to use `curl -fsSL` for scripts as it's more commonly available and has more robust error handling options.